labs/10.md

```diff
@@ -1,192 +0,0 @@
----
```

Comment:
Why are the labs removed in this pull request instead of being moved in a426dfb9fd4ebac1f3993eca0bc274b2c3bdb141 into the archive?

Reply:
The rendered HTML of the labs is in the archives, I just removed the markdown files. Unless you mean something else?

Reply:
Oh, I just expected the whole contents minus the `archives` directory to be archived in that commit, so the labs and stuff would have been moved too (and removed from here I suppose), but this works fine too.